### PR TITLE
CON-1977 content and design changes to create an advert second half

### PR DESCRIPTION
**Pull request by ben1stone**
Updates content and design on the Pay, working hours and preview start page.

The updated VacacnyServices.Wage package was to change the content that is returned from the GetYearlyMinimumWageText method

### CODE REVIEW

File: src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml [hunk 1]
```diff
@@ -32,7 +32,7 @@
                     <span esfa-validation-message-for="Duration" class="govuk-error-message"></span>
                     <input asp-for="Duration" class="govuk-input govuk-input--width-2" type="number" min="1" >
                     <select asp-for="DurationUnit" class="govuk-select duration-unit-select">
-                        <option value="@DurationUnit.Month">Month(s)</option>
+                        <option value="@DurationUnit.Month">Months</option>
                         <option value="@DurationUnit.Year">Year(s)</option>
```
Comment (shomavg) [inline src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml:36]: why not make the same change for Years too?

File: src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj [hunk 2]
```diff
@@ -37,7 +37,7 @@
     <PackageReference Include="RestSharp" Version="106.6.10" />
     <PackageReference Include="SFA.DAS.Account.Api.Client" Version="1.6.1601" />
     <PackageReference Include="SFA.DAS.Http" Version="1.2.4" />
-    <PackageReference Include="SFA.DAS.VacancyServices.Wage" Version="1.0.8" />
+    <PackageReference Include="SFA.DAS.VacancyServices.Wage" Version="1.0.13" />
```
Comment (shomavg) [inline src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj:40]: why is the wage updated?

Comment (ben1stone) [reply src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj:40]: The updated VacacnyServices.Wage package was to change the content that is returned from the GetYearlyMinimumWageText method

File: src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml (hunk 1 above)
Comment (ben1stone) [reply src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml:36]: Apprenticeships are minimum 12 Months long so the optional (s) doesn't make sense but they could choose to write that as 1 Year.

File: src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj (hunk 2 above)
Comment (shomavg) [reply src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj:40]: 👍

File: src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml (hunk 1 above)
Comment (shomavg) [reply src/Employer/Employer.Web/Views/Part1/Duration/Duration.cshtml:36]: 👍

**Pull request merged**